### PR TITLE
chore: add build profiling info in readme

### PR DESCRIPTION
**Pull request by pravusjif**
README changes:
* Added build profiling info
* Added xcrun error info
* Fixed "manual build" section info

### CODE REVIEW

**Review by sandrade-dcl — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -204,26 +209,43 @@ public IEnumerator VisualTestStub_Generate()
 
 ### Making a manual build
 
-1. Build unity WASM with its name as `unity` into the folder `/static`. **It's very important that the folder/build name is `unity`**.
-2. Checkout the file named `static/unity/Build/DCLUnityLoader.js`. Unity deletes anything on this folder as part of the build process and we need that.
+#### Option 1
+1. Run `make watch` in a terminal at `kernel/static`. Wait for it to finish and leave it watching for changes. **This step must be done before replacing the build files or they may be replaced again with the latest published build files when the `make watch` process runs**
```
Comment (pbosio) [inline README.md]: `make watch` shouldn't be done at directory `kernel/static` I find this kind of misleading

File: README.md [hunk 2]
```diff
@@ -204,26 +209,43 @@ public IEnumerator VisualTestStub_Generate()
...
 
-5. Run `make watch`.
 6. Testing how your new build performs:
 
 - Open **[http://localhost:8080/?DEBUG_MODE&LOCAL_COMMS&position=-100,100](http://localhost:8080/?DEBUG_MODE&LOCAL_COMMS&position=-100,100)** to go to an area with a high density of test parcels.
 - Open **[http://localhost:8080/?DEBUG_MODE&LOCAL_COMMS&ENV=org&position=10,0](http://localhost:8080/?DEBUG_MODE&LOCAL_COMMS&ENV=org&position=10,0)** to open an area with real-life deployments (but without communicating with other users).
 - Open **[http://localhost:8080/?ENV=org&position=0,90](http://localhost:8080/?ENV=org&position=0,90)** to open the explorer near the Decentraland Museum
 
+#### Option 2
+Alternatively you can build into any other folder in your disk and then just move the 3 ".unityweb" files that will end up inside `[ChosenDestination]/unity/Build/` into `kernel/static/unity/Build/`. Just remember that if you run the `make watch` after this, your new files may be replaced by the latest published build files during the `make watch` process.
```
Comment (pbosio) [inline README.md]: Can we make this Option 1? cause current Option 1 is over-complicated

Comment (pravusjif) [reply README.md]: totally

File: README.md (hunk 1 above)
Comment (pravusjif) [reply README.md]: totally XD

Comment (pravusjif) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (pravusjif) [reply README.md]: Done

**Review by pbosio — APPROVED** [review]

**Pull request merged**